code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/process/ServicesReconcilePostTrigger.java

```diff
@@ -0,0 +1,52 @@
+package io.cattle.platform.servicediscovery.process;
+
+import io.cattle.platform.archaius.util.ArchaiusUtil;
+import io.cattle.platform.core.model.Service;
+import io.cattle.platform.engine.handler.HandlerResult;
+import io.cattle.platform.engine.handler.ProcessPostListener;
+import io.cattle.platform.engine.process.ProcessInstance;
+import io.cattle.platform.engine.process.ProcessState;
+import io.cattle.platform.process.common.handler.AbstractObjectProcessLogic;
+import io.cattle.platform.servicediscovery.deployment.DeploymentManager;
+import io.cattle.platform.servicediscovery.service.ServiceLookup;
+
+import java.util.Collection;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import com.netflix.config.DynamicStringListProperty;
+
+@Named
+public class ServicesReconcilePostTrigger extends AbstractObjectProcessLogic implements ProcessPostListener {
+    private static final DynamicStringListProperty PROCESS_NAMES = ArchaiusUtil
+            .getList("service.reconcile.posttrigger.processes");
```

Comment:
Since I don't foresee this value changing often (for example, between prod vs debug or for other types of environments), I'm not sure if there's that much value extracting it out like this.  Also, funny enough, I've actually grown to like referencing the *Constants classes we've created.  I've noticed it's actually a pretty easy way to find all the handlers for that particular event.  For example, in eclipse, I could just look for all references to HostConstants.PROCESS_CREATE.